### PR TITLE
PRESIDECMS-2128 datamanager add one to many action customization

### PR DESCRIPTION
**Pull request by karhong**

### CODE REVIEW

File: system/handlers/admin/DataManager.cfc [hunk 1]
```diff
@@ -2423,6 +2423,16 @@ component extends="preside.system.base.AdminHandler" {
 
 		validationResult = validateForm( formName=arguments.formName, formData=formData, stripPermissionedFields=arguments.stripPermissionedFields, permissionContext=arguments.permissionContext, permissionContextKeys=arguments.permissionContextKeys );
 
+		args.formData         = formData;
```
Comment (DominicWatson) [inline system/handlers/admin/DataManager.cfc:2427]: `args` has not been initialized here. ie, we should probably have it in the arguments list ( `struct args = {}`)

**Review by DominicWatson — CHANGES_REQUESTED** [review]

**Comment by DominicWatson** [comment]
Looking good, just one little thing to feed back on.

File: system/handlers/admin/DataManager.cfc (hunk 1 above)
Comment (karhong) [reply system/handlers/admin/DataManager.cfc:2427]: I'm initialized via `var args = arguments;` by following how the DataManager._addRecordAction did instead. Might also useful to have more params passing to customization point

**Comment by DominicWatson** [comment]
Nice one, merged into upcoming release.

**Pull request merged**